tests/installation/partitioning_raid.pm

```diff
@@ -148,45 +148,71 @@ sub run() {
 
     # user defined
     send_key $cmd{custompart};
+    assert_screen 'custompart_option-selected';
     send_key $cmd{next};
     assert_screen 'custompart';
 
     send_key "tab";
-    send_key "down";    # select disks
+    assert_screen 'custompart_systemview-selected';
+    send_key "down";
+    assert_screen 'partitioning_raid-hard_disks-selected';
+
     if (get_var("OFW")) {    ## no RAID /boot partition for ppc
+        wait_idle 3;
```

Comment:
Let's make it more clear: Don't add `wait_idle`, remove them where you can -> https://progress.opensuse.org/issues/5830
Use `wait_still_screen`, `wait_screen_change` or `sleep` instead